homework6/src/Ex5.java

```diff
@@ -0,0 +1,45 @@
+import java.util.Scanner;
+
+public class Ex5 {
+
+    public static boolean isAnagram(String s1, String s2){
+        if(s1.length() != s2.length())
+            return false;
+
+        int[] s1CharFrequency = new int[123]; //puteam folosi un hashmap dar avand in vedere ca sunt cuvinte overhead-ul pentru declararea unui hashmap ar fi fost prea mare
```

Comment:
de ce 123? daca pui cuvintele {sam|ana} si {ana|sam} ce se va intampla?

Reply:
am citit rapid cerinta si credeam ca se referea ca functia sa verifice doar cuvinte si asa ca am pus 123 deoarece in ascii ultima litera din alfabet este 122. Acum ca am recitit vad ca se puteau folosi toate caracterele

Reply:
de regula nu as fi spus nimic deoarece e ft ok! Insa stiu ca esti inzestrat si atunci imi place sa te provoc deoarece stiu ca intelegi lucrurile mai rapid si mai profund decat majoritatea!

Reply:
Multumesc mult ! Chiar vreau sa fiu provocat pentru ca scopul meu cu acest curs este sa inteleg cat mai bine limbajul si sa dobandesc cat mai multe cunostinte.